### ISSUE TITLE
szczawnica-belweder.pl

### ISSUE THREAD

**Issue by felix-22** [MajkiIT/polish-ads-filter#19089]
Czy nie da się tego jakimś generykiem, bo sporo jest takich komunikatów:

![screenshoot_125874](https://user-images.githubusercontent.com/75935068/111083805-92460c80-850f-11eb-8d77-2d9f0b4fbd02.png)

**Comment by strict-block** [comment]
Nie, sprawdzanie tekstu jest lekko za wolne w porównaniu do `:upward()` bazujących na linku w pop-upie. Dodatkowo będzie raczej do końca świata wyciekać w ABP jak ci nie uruchomią generyków proceduralnych i najlepiej `:upward()`

Więc ostatecznie jest takie coś pod domeny:

`#?#.widget.type-10:not([id]):-abp-contains(/c(ookie|iastecz)/i)`, bo schemat z `<aside>` lub `<section>` przestał wyłapywać tylko cookies.

Może kiboke dodał by generyczny skrypt co by klikał zamknij, pewnie ze szkodami zamykania np. newsletterów.
